packages/create-amplify/src/get_project_root.ts

```diff
@@ -27,7 +27,13 @@ export const getProjectRoot = async () => {
   if (!isExistProjectRoot) {
     logger.debug(`The provided directory (${projectRoot}) does not exist.`);
     logger.debug(`Creating directory ${projectRoot}`);
-    await fsp.mkdir(projectRoot, { recursive: true });
+    try {
+      await fsp.mkdir(projectRoot, { recursive: true });
+    } catch (e) {
+      await fsp.mkdir(path.join(process.cwd(), projectRoot), {
+        recursive: true,
+      });
```

Comment:
we should not do this.
When `/` is provided at the beginning of the path the intent is that path is absolute. Therefore we shouldn't alter it like that, but rather figure out if error we get is expected one and whether it could be presented in better way.

Reply:
Perhaps we just need an if in the catch to print some additional info for absolute dirs:
```ts
} catch (e) {
  if (path.isAbsolute(projectRoot) {
    logger.warn(`Failed to create directory at ${projectRoot}. Ensure this is the correct path and you have write permissions to this location.`)
  }
  throw e
}
```

Reply:
thank you feedback!!!!
later fix code